.github/workflows/deepx-ci.yml

```diff
@@ -0,0 +1,37 @@
+name: DeepX CI
+
+run-name: Build Docker container and perform unit testing
+
+on: 
+  push:
+    branches: [gz-rendering7]
+  pull_request:
+    branches: [gz-rendering7]
+  workflow_dispatch:
+
+jobs:
+  build-library:
+    runs-on: ubuntu-20.04
+    name: Build and test
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+
+      - name: Login to Github Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: deepx-inc
+          password: ${{ github.token }}
+
+      - name: Build the Docker image
+        run: |
+          docker build -t deepx-gz-rendering .
+      
+      # Skip ctest for now because many tests are throwing exceptions due to lack of display.
+      - name: Build gz-rendering and test
+        run: |
+          docker run deepx-gz-rendering:latest bash \
+          -c 'cd /root/gazebo/gz-rendering/build; \
+          cmake ..; \
+          cmake --build . --target install;'
```

Comment:
Sorry for nitpicking. A blank line before the EOF is missing. Could you please add one? Thank you!